src/backend/database_models/agent.py

```diff
@@ -0,0 +1,41 @@
+from enum import StrEnum
+
+from sqlalchemy import Enum, Float, Integer, String, Text, UniqueConstraint
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from backend.database_models.base import Base
+
+
+class Deployment(StrEnum):
+    COHERE_PLATFORM = "Cohere Platform"
+    SAGE_MAKER = "SageMaker"
+    AZURE = "Azure"
+    BEDROCK = "Bedrock"
+
+
+class Model(StrEnum):
```

Comment:
Some deployments have different names for the models, so we might need to have some sort of mapping. It can be a follow up tho.

Reply:
true. let me add a TODO note